example/package.json

```diff
@@ -18,9 +18,6 @@
   "devDependencies": {
     "@babel/plugin-syntax-object-rest-spread": "^7.8.3"
   },
-  "eslintConfig": {
-    "extends": "react-app"
-  },
```

Comment:
why are you suggesting to remove the eslint extension in the example project?

Reply:
fixed and re-added. @giacomocerquone